lib/solr/query/request/runner.rb

```diff
@@ -22,10 +22,7 @@ def initialize(page:, page_size:, solr_params: {})
 
         def run
           raw_response = connection(PATH).post_as_json(request_params)
-          response = Solr::Response.from_raw_response(raw_response)
-          Solr.instrument(name: 'solrb.request_response_cycle',
-                          data: { request: request_params, response: raw_response })
-          response
```

Comment:
@Valve This instrumentation is needed for testing:
https://github.com/machinio/solrb/blob/master/lib/solr/testing.rb

Reply:
@drnluz oh, I didn't realize it, should I return it?
Let's document this in the readme

Reply:
@Valve yes because without this we won't be able to run some specs on machinio

Reply:
Got it, I'll return, please add this to readme when you get a chance